etc/pybitmessage.profile

```diff
@@ -0,0 +1,53 @@
+# Firejail profile for default
+# This file is overwritten after every install/update
+# Persistent local customizations
+include /etc/firejail/pybitmessage.local
+
+# generic gui profile
+# depending on your usage, you can enable some of the commands below:
+noblacklist /sbin
+noblacklist /usr/local/sbin
+noblacklist /usr/sbin
```

Comment:
I don't see the reason for those three. Delete them.

Reply:
I get the error and crush pybitmessage start if I comment all or any this three lines.
```
Traceback (most recent call last):
  File "bitmessagemain.py", line 19, in <module>
    import depends
  File "/usr/share/pybitmessage/depends.py", line 5, in <module>
    import pyelliptic.openssl
  File "/usr/share/pybitmessage/pyelliptic/__init__.py", line 16, in <module>
    from .openssl import OpenSSL
  File "/usr/share/pybitmessage/pyelliptic/openssl.py", line 549, in <module>
    loadOpenSSL()
  File "/usr/share/pybitmessage/pyelliptic/openssl.py", line 547, in loadOpenSSL
    raise Exception("Couldn't find and load the OpenSSL library. You must install it.")
Exception: Couldn't find and load the OpenSSL library. You must install it.
```
```
/etc/firejail/disable-common.inc:blacklist /sbin
/etc/firejail/disable-common.inc:blacklist /usr/local/sbin
/etc/firejail/disable-common.inc:blacklist /usr/sbin
```

Reply:
Does it work if you add "openssl*" to private-bin option?

Reply:
No, I checked it does not work. The same error occurs.